src/coreclr/src/System.Private.CoreLib/src/System/Math.CoreCLR.cs

```diff
@@ -11,119 +11,154 @@
 ===========================================================*/
 
 using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
 
 namespace System
 {
     public static partial class Math
     {
+#pragma warning disable CA1401 // P/Invokes should not be visible
+
         [Intrinsic]
-        [MethodImpl(MethodImplOptions.InternalCall)]
+        [SuppressGCTransition]
+        [DllImport(RuntimeHelpers.QCall, CallingConvention = CallingConvention.Cdecl)]
         public static extern double Abs(double value);
 
+        [SuppressGCTransition]
+        [DllImport(RuntimeHelpers.QCall, CallingConvention = CallingConvention.Cdecl)]
+        private static extern float AbsF(float value);
```

Comment:
`EntryPoint` does not get registered with QCalls; as in:

```c#
[DllImport(RuntimeHelpers.QCall, CallingConvention = CallingConvention.Cdecl, EntryPoint = "AbsF")]
public static float Abs(float value);
```
with:
```c++
QCFuncElement("AbsF", fabsf)
```
it continues to match by method name, and silently uses double `QCFuncElement("Abs", fabs)`, which later results in test failures. Changed this one to call private method.